tests/README.VCRTesting.md

```diff
@@ -0,0 +1,33 @@
+# VCR Testing
+
+VCR are tests run utilizing the go library go-vcr to replay recorded http
+interactions. This enables test runs for PR pre-submit as resources aren't
+actually being created/deleted and no real GCP interactions are made, allows for
+efficient and reliable testing of KCC resources.
+
+VCR is controlled via the environment variables E2E_GCP_TARGET and VCR_MODE. To
+run VCR test, E2E_GCP_TARGET need to be set to `vcr`.
+
+## Record Mode
+We pre-captured vcr logs by set VCR_MODE to `record`. Test will still run
+against real GCP, logs, or cassettes will be stored under each test folder, e.g.
```

Comment:
I'm not sure what this means.  Maybe "In this mode, tests runs against real GCP, but casettes are created based on the actual API calls."

You might also then introduce "casettes" in the previous section, e.g. by adding a 3rd paragraph like "This functionality is implemented using the go library go-vcr.  go-vcr introduces a videotape metaphor: first you record the actual GCP API calls in a "casette", then you can run the tests again without GCP API calls by "replaying" the casette"

Reply:
I updated to use "vcr logs" instead of using "cassettes", to prevent confusions from users that may not have go-vcr context.